README.md

```diff
@@ -114,6 +114,16 @@ def info(...)
 * `scipy.io` or `soundfile` as default for __wav__ files
 * `aubio` or `audioread` for __mp3__ and __mp4__
 
+
+### Installation
+
+`pip install .`
```

Comment:
since we are all devs for now, lets make an editable install... `pip install -e .`

Reply:
Sure. That sounds good. I made the change and the PR should automatically include this change.